src/ngraph/frontend/onnx_import/utils/broadcasting.cpp

```diff
@@ -17,12 +17,86 @@
 #include <numeric>
 #include <vector>
 
+#include "ngraph/op/broadcast.hpp"
+#include "ngraph/op/reshape.hpp"
+
 #include "broadcasting.hpp"
 
+static std::vector<ngraph::Shape> calculate_numpy_broadcast_shape(ngraph::Shape left_shape,
```

Comment:
Add some documentation. What is this function supposed to do?
You can provide a link to numpy documentation.